File: Javascript/26. Remove Duplicates from Sorted Array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
 var removeDuplicates = function(nums) {

    let curr = 0;
    for(let i=1;i<nums.length;i++){
        if(nums[curr]==nums[i]) continue;
        nums[++curr] = nums[i];
    }

    return ++curr;
    
};

File: Java/26. Remove Duplicates from Sorted Array.java
package Java;
class Solution {
    public int removeDuplicates(int[] nums) {
        int curr = 0;
        for(int i=1;i<nums.length;i++){
            if(nums[curr]==nums[i]) continue;
            nums[++curr] = nums[i];
        }

        return ++curr;
        
    }
}